statvar_imports/us_crash/fars_crashdata/download_script.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 20 ('License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#           https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# How to run the script to download the files:
+# python3 download_script.py
+
+import os
+import requests
+import shutil
+import sys
+import zipfile
+
+from absl import app
+from absl import logging
+import download_config
+_SCRIPT_PATH = os.path.dirname(os.path.abspath(__file__))
+GCS_OUTPUT_DIR = os.path.join(_SCRIPT_PATH, 'gcs_output')
+if not os.path.exists(GCS_OUTPUT_DIR):
+            os.makedirs(GCS_OUTPUT_DIR)
+sys.path.append(os.path.join(_SCRIPT_PATH, '../../../util/')) 
+from download_util_script import _retry_method
+
+
+def download_and_extract_data(url, input_folder, year):
+    """Downloads zip, extracts ACCIDENT.CSV (handling nested dirs), with retry"""
+    try:
+        response = _retry_method(url, None, 3, 5, 2)
+        filename = os.path.basename(url)
+        input_folder = os.path.join(GCS_OUTPUT_DIR, input_folder)
+        filepath = os.path.join(input_folder, filename)
+
+        if not os.path.exists(input_folder):
+            os.makedirs(input_folder)
```

Comment:
Add exist_ok=True to os.makedirs(GCS_OUTPUT_DIR) to prevent  FileExistsError if the directory already exists.

Reply:
added